tooling/version.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "Build Packages and Script"
+npm run build
+npm run build:script
+
+echo "Bump all package version"
+npm run tooling -- version set "${npm_package_version}" --all
+
+echo "Generate Changelog"
+npm run changelog:generate
+
+git add packages/node_modules/*/package.json packages/node_modules/@ciscospark/*/README.md packages/node_modules/@webex/*/package.json packages/node_modules/webex/umd/*.js CHANGELOG.md
```

Comment:
Why are you still updating `@ciscospark`?

Reply:
Can you move this command to the `Push to upstream/master` step instead?
It's weird (and prone to cause future confusion) that git staging happens here and the other git stuff happens elsewhere. I'd prefer if you can do all the `git` stuff together

Reply:
@ciscospark is a left over from during the rename/new pipeline changes (and doesn't hurt to keep since it's not doing anything)

Reply:
the `git add` is needed since `npm version` will only commit the `package.json`

https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-cli#with-npm-version

Reply:
Can you add a comment that points to this then?

Reply:
> @ciscospark is a left over from during the rename/new pipeline changes (and doesn't hurt to keep since it's not doing anything)
> I could just change it to `git add -A`

Whichever one is fine